### ISSUE TITLE
TypeError

### ISSUE THREAD

**Issue by OliveS9** [kingoflolz/mesh-transformer-jax#161]
Could anyone please fix the following import error when running "from mesh_transformer.checkpoint import read_ckpt_lowmem"? Thanks a lot.

"TypeError: Cannot subclass <class 'typing._SpecialForm'>"

**Comment by Aspie96** [comment]
I have this issue too.

**Comment by Aspie96** [comment]
I believe the issue is caused by this commit: https://github.com/deepmind/optax/commit/f777e5ce8df4fc78b1f98fdfc5abe813da5ceef6

Please, @OliveS9, try downgrading optax to 0.0.9:
```
!git clone https://github.com/kingoflolz/mesh-transformer-jax.git
!pip install -r mesh-transformer-jax/requirements.txt
!pip install optax==0.0.9 transformers dm-haiku einops
!pip install ray
```

**Comment by OliveS9** [comment]
Thank you very much.

**Comment by OliveS9** [comment]
@Aspie96  

Thank you again for your previous help. Do you know how to fix the following FileNotFoundError? 

"FileNotFoundError: [Errno 2] No such file or directory: 'step_383500/shard_0/0.npz'"

Thanks a lot.

**Comment by kingoflolz** [comment]
Fixed in 628f44dcb9e9dbc94e46e47339a95f2f3b4694f9
